src/pages/LoginSignupPage/LoginSignupPage.scss

```diff
@@ -0,0 +1,39 @@
+* {
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans',
+    'Helvetica Neue', sans-serif;
+  box-sizing: border-box;
+}
+
+.loginSignupPage__container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  height: 100vh;
+  background: linear-gradient(0deg, rgba(40, 103, 178, 0.28895308123249297) 0%, rgba(255, 255, 255, 1) 71%);
+  .loginSignupPage__signinText {
+    margin-bottom: 1.5rem;
+  }
+}
+.cta__button {
+  border: none;
+  background: none;
+  color: rgb(76, 76, 212);
```

Comment:
store and get fonts from Fonts.scss
https://github.com/Billennium-frontend-interns/linkedin_clone_project/pull/13/files